common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -107,17 +107,26 @@ private static class WynncraftButton extends Button {
         }
 
         @Override
-        public void renderWidget(PoseStack matrices, int mouseX, int mouseY, float partialTicks) {
-            super.renderWidget(matrices, mouseX, mouseY, partialTicks);
+        public void renderWidget(GuiGraphics guiGraphics, int mouseX, int mouseY, float partialTicks) {
+            super.renderWidget(guiGraphics, mouseX, mouseY, partialTicks);
 
             if (serverIcon == null || serverIcon.getServerIconLocation() == null) {
                 return;
             }
 
-            RenderSystem.setShaderTexture(0, serverIcon.getServerIconLocation());
-
             // Insets the icon by 3
-            blit(matrices, this.getX() + 3, this.getY() + 3, this.width - 6, this.height - 6, 0, 0, 64, 64, 64, 64);
+            guiGraphics.blit(
```

Comment:
In the previous commit, you changed a blit to a call to RenderUtils in the overlay management screen. I assumed the method were no longer accessible, but maybe there was another reason..?

Reply:
That blit method was incompatible with GuiGraphics, and I rather converted it than fix it (also it was in a screen so that would just make it normal). This case, it was perfectly fine, and this is kind of an edge-case widget. So I left it.